packages/styled-ui-docs/pages/scrollbar.mdx

```diff
@@ -260,17 +261,18 @@ render(
 
 | Name | Type | Default | Description |
 | :--- | :--- | :------ | :---------- |
-| alwaysShow | boolean | false | Enable always-show mode. |
-| autoHideDelay | number | 1000 | Hide delay in ms. |
-| autoHeight | boolean | false | Enable auto-height mode. |
-| autoHeightMax | number | 200 | Set a maximum height for auto-height mode. default is `200px`. |
-| autoHeightMin | number | 0 | Set a minimum height for auto-height mode. |
+| horizontalVisibility | string \| boolean | 'auto' | Acceptable values: `'auto'`, `true`, `false`. If `'auto'`, the horizontal scrollbar will auto show/hide. If `true`, the horizontal scrollbar always to show.  If `false`, the horizontal scrollbar always to hide. |
+| verticalVisibility | string \| boolean | 'auto' | Acceptable values: `'auto'`, `true`, `false`. If `'auto'`, the vertical scrollbar will auto show/hide. If `true`, the vertical scrollbar always to show.  If `false`, the vertical scrollbar always to hide. |
+| autoHideDelay | number | 1000 | Hide delay in ms. |transitionDelay
```

Comment:
Remove the trailing "transitionDelay"

Reply:
@tinaClin  fixed in 4e373f073231380b90be7eff227a8b7d2891b212